### PR TITLE
i18n(fr) Add French translation for `content.mdx` `talks.mdx` and update `recipes.mdx` and `nav.ts`

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### Description (required)

Add French translation for `content.mdx` `talks.mdx` and update `recipes.mdx` and `nav.ts`

PR #5614 #6470

### CODE REVIEW

File: src/content/docs/fr/recipes.mdx [hunk 1]
```diff
@@ -15,27 +15,6 @@ Voir des exemples guidés pour ajouter des fonctionnalités à votre projet Astr
...
-- [Mise en place de micro-frontaux avec Astro](https://medium.com/@sergio.a.soria/setting-up-micro-frontends-with-astro-and-ecma-script-modules-137340d2c520)
-- [Ajouter l'authentification par nom d'utilisateur et mot de passe avec Lucia](https://lucia-auth.com/guidebook/sign-in-with-username-and-password/astro)
-- [Ajouter GitHub OAuth avec Lucia](https://lucia-auth.com/guidebook/github-oauth/astro)
-- [Intégration de Sentry avec Astro](https://akashrajpurohit.com/blog/seamless-error-tracking-integrating-sentry-with-astro/)
-- [Configurer efficacement les pages de brouillon dans Astro avec la création de contenu basée sur la configuration](https://akashrajpurohit.com/blog/set-up-draft-pages-effectively-in-astro-with-configdriven-content-authoring/)
-- [Comment envoyer n'importe quelle donnée vers votre flux RSS ? Un guide avec Astro + RSS](https://aritraroy.live/tutorial/blogs/2023/how-to-send-any-data-to-rss-feed/)
+
+
+<ReadMore>Vous trouverez encore plus de recettes rédigées et mises à jour par la communauté dans nos [recettes et guides de la communauté]](/fr/community-resources/content/#recipes-and-guides)</ReadMore>
```
Comment (thomasbnt) [inline src/content/docs/fr/recipes.mdx:20]:
```suggestion
<ReadMore>Vous trouverez encore plus de recettes rédigées et mises à jour par la communauté dans nos [recettes et guides de la communauté](/fr/community-resources/content/#recipes-and-guides)</ReadMore>
```

**Review by thomasbnt — CHANGES_REQUESTED** [review]
One link to fix, and LGTM!

**Comment by McFlyPartages** [comment]
> One link to fix, and LGTM!

Thanks it's ok

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Comment by thomasbnt** [comment]
Owh, miss CI Links and good!

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Comment by VoxelMC** [comment]
Thanks!!

`As a non-native speaker, I reviewed this PR using the Docs Translation Review Checklist and it LGTM!`

**Review by dreyfus92 — APPROVED** [review]
Merci les gars, LGTM 😁

**Pull request merged**